src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,14 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(n -> Arrays.stream(n.split(",")))
+                .mapToInt(Integer::parseInt)
+                .filter(i -> i % 2 == 0)
+                .min()
+                .orElseThrow(() -> new RuntimeException(
+                        "Can't get min value from list: < Here is our input 'numbers' >"));
```

Comment:
```suggestion
                        "Can't get min value from list: " + numbers));
```
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/java-eight-part-two/stream-api#in-method-findminevennumber-exception-message-should-look-like-this)